lib/commonConfig.js

```diff
@@ -451,10 +450,6 @@ function processEnvironmentVariables() {
         config.pollingDaemonFrequency = process.env.IOTA_POLLING_DAEMON_FREQ;
     }
 
-    if (process.env.IOTA_AUTOCAST) {
-        config.autocast = process.env.IOTA_AUTOCAST === 'true';
-    }
-
```

Comment:
This change seems similar to the ones in PR https://github.com/telefonicaid/iotagent-node-lib/pull/1501. Is this PR based on PR #1501? Or it is just it has touched the same code but without direct relationship?

Whichever the case, it should include proper CHANGES_NEXT_RELEASE entry:

```
- Remove: autocast (including env var IOTA_AUTOCAST) (#1498)
```